.github/SECURITY.md

```diff
@@ -0,0 +1,22 @@
+# Responsible Disclosure Policy
+
+At the IOTA Foundation, we consider the security of our systems a top priority. Nevertheless, no matter how much effort we put into system security, there can still be vulnerabilities present. If you've discovered a vulnerability, please follow the guidelines below to report it to our security team:
+
+- E-mail your findings to [security@iota.org](mailto:security@iota.org). If the report contains highly sensitive information, please consider encrypting your findings using our [security@iota.org](mailto:security@iota.org) (1F211CB6E8A158722F9053D3E27A933040CF05B9) PGP key.
+
+## Rules for Testing and Reporting Vulnerabilities:
+
+- Do not take advantage of the vulnerability you have discovered, for example by downloading more data than is necessary to demonstrate the vulnerability. 
+- Do not read, modify or delete data that you don't own.
+- Do not disclose the problem to third parties until it has been resolved.
+- The scope of the program is limited to technical vulnerabilities in IOTA Foundations' web applications and open source software packages distributed through GitHub. Please do not try to test physical security or attempt phishing attacks against our employees, and so on.
+- Out of concern for the availability of our services to all users, please do not attempt to carry out DoS attacks, leverage black hat SEO techniques, spam people, or other similarly questionable things. We also discourage the use of any vulnerability testing tools that automatically generate significant volumes of traffic.
+
+## What We Promise:
+
+- We will respond to your report within 3 business days with our evaluation of the report, and an expected resolution date.
+- If you have followed the instructions above, we will not take any legal action against you in regard to the report.
+- We will keep you informed during all stages of resolving the problem.
+- To show our appreciation for your effort and cooperation during the report, we will list your name, and a link to a personal website/social network profile on the page below.  This way the public can know you've helped keep the IOTA Foundation secure.
```

Comment:
Specify that we will only do this if they want us to?

Reply:
Done